scalajvm/app/global/Options.scala

```diff
@@ -0,0 +1,10 @@
+package global
```

Comment:
I meant that to be the "global" handler for all HTTP `OPTIONS` requests. But

1) it's not as global anymore, because it handled only `/api` route
2) I agree; moved it to `cors` package